Modules/arraymodule.c

```diff
@@ -1544,7 +1544,7 @@ array_array_fromlist(arrayobject *self, PyObject *list)
         if (array_resize(self, old_size + n) == -1)
             return NULL;
         for (i = 0; i < n; i++) {
-            PyObject *v = PyList_GetItem(list, i);
+            PyObject *v = PyList_GET_ITEM(list, i);
```

Comment:
It is possible the the list change size when convert its items to C integers. `PyList_GetItem()` can return NULL.

Needed to check either that v != NULL, or that n == PyList_GET_SIZE() in a loop.

Reply:
Sorry. As this is a bug in the existing code, can I simply revert this change?

Reply:
It is up to you. Either revert this change and create a separate PR for fixing this bug, or include a fix for it in this PR.